docs/source/guide/pt-1-intro.md

```diff
@@ -0,0 +1,101 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use PT?
+
+As with all techniques, PT is compatible with any frontend supported by Mitiq:
+
+```{code-cell} ipython3
+import mitiq
+
+mitiq.SUPPORTED_PROGRAM_TYPES.keys()
+```
+
+
+## Problem setup
+We first define the circuit of interest. In this example, the circuit has 
+two CNOT gates and a CZ gate. We can see that when we apply Pauli Twirling,
+we will generate 
+
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, CZ, CNOT
+
+a, b, c, d  = LineQubit.range(4)
+circuit = Circuit(
+    CNOT.on(a, b),
+    CZ.on(b, c),
+    CNOT.on(c, d),
+)
+
+print(circuit)
+```
+
+Next, we use introduce noise into the input circuit and then use
+{func}`.pauli_twirl_circuit()` to tailor the noise in the noisy
+circuit. It is worth noting that this technique skips defining an executor
+as the end goal is to tailor the noise rather than estimate the expectation
+value of some observable. 
+
+
+
+```{code-cell} ipython3
+import numpy as np
+from cirq import amplitude_damp
+from mitiq.interface import convert_to_mitiq
+
+noise_level = 0.1
+mitiq_circuit, _ = convert_to_mitiq(circuit)
+noisy_circuit = mitiq_circuit.with_noise(amplitude_damp(gamma=noise_level))
+
+```
+
+```{admonition} Note:
+PT is designed to transform the noise simulated in this example,
+but it should not be expected to always be a positive effect in terms of
+obtaining a better expectation value. Thus, as discussed previously,
+it is more of a noise tailoring technique, designed
+to be composed with other error mitigation techniques rather than an 
+error mitigation technique on its own.
+```
+
+## Apply PT
+Pauli Twirling can be easily implemented with the function
+{func}`.pauli_twirl_circuit()`. As an example, the original input circuit without
+noise inserted into it is used here.
+
+```{code-cell} ipython3
+from mitiq import pt
+tailored_circuit_ideal = pt.pauli_twirl_circuit(
+    circuit = circuit,
+    num_circuits = 11,
+)
+```
+The default option for prividing different twirled versions of the same input circuit
+is set to 10. For the choice above, the twirled outputs are as shown below. 
+
+```{code-cell} ipython3
+for i in range(len(tailored_circuit_ideal)):
+    print(f"Example Pauli Twirled Circuit {i}: \n {tailored_circuit_ideal[i]}, \n")
+```
+
+Here, we observe that the application of the Pauli Twirling technique has kept the logic
+of the input circuit unchanged while introducing a combination of gates to twilred outputs.
+
+## Combine PT with ZNE
```

Comment:
Move Aaron's code here where the ideal expectation value is compared to the mitigated one.